internal/test_helpers/fixtures/bash/history_persistence/pass

```diff
@@ -0,0 +1,162 @@
+Debug = true
+
+[33m[stage-5] [0m[94mRunning tests for Stage #5: za2[0m
+[33m[stage-5] [0m[94mRunning ./your_shell.sh[0m
+[33m[stage-5] [0m[94mHistory file content:[0m
+[33m[stage-5] [0m[pear_shell_history_test] echo blueberry mango
+[33m[stage-5] [0m[pear_shell_history_test] echo pineapple grape
+[33m[stage-5] [0m[pear_shell_history_test] echo grape apple
+[33m[your-program] [0m$ echo mango orange pear
+[33m[your-program] [0mmango orange pear
+[33m[stage-5] [0m[92m✓ Ran echo mango orange pear[0m
+[33m[your-program] [0m$ echo raspberry banana
+[33m[your-program] [0mraspberry banana
+[33m[stage-5] [0m[92m✓ Ran echo raspberry banana[0m
+[33m[your-program] [0m$ echo raspberry strawberry
+[33m[your-program] [0mraspberry strawberry
+[33m[stage-5] [0m[92m✓ Ran echo raspberry strawberry[0m
+[33m[your-program] [0m$ history
+[33m[your-program] [0m    1  echo mango orange pear
+[33m[your-program] [0m    2  echo raspberry banana
+[33m[your-program] [0m    3  echo raspberry strawberry
+[33m[your-program] [0m    4  history
+[33m[stage-5] [0m[92m✓ History before loading file is correct[0m
+[33m[your-program] [0m$ history -r /tmp/pear_shell_history_test
+[33m[stage-5] [0m[92m✓ history -r command executed[0m
+[33m[your-program] [0m$ history
+[33m[your-program] [0m    1  echo mango orange pear
+[33m[your-program] [0m    2  echo raspberry banana
+[33m[your-program] [0m    3  echo raspberry strawberry
+[33m[your-program] [0m    4  history
+[33m[your-program] [0m    5  history -r /tmp/pear_shell_history_test
+[33m[your-program] [0m    6  echo blueberry mango
+[33m[your-program] [0m    7  echo pineapple grape
+[33m[your-program] [0m    8  echo grape apple
+[33m[your-program] [0m    9  history
+[33m[stage-5] [0m[92m✓ History after loading file is correct[0m
+[33m[your-program] [0m$ 
+[33m[stage-5] [0m[92mTest passed.[0m
+
+[33m[stage-4] [0m[94mRunning tests for Stage #4: in3[0m
+[33m[stage-4] [0m[94mRunning ./your_shell.sh[0m
+[33m[your-program] [0m$ echo orange raspberry
+[33m[your-program] [0morange raspberry
+[33m[stage-4] [0m[92m✓ Received expected response[0m
+[33m[your-program] [0m$ echo apple banana
+[33m[your-program] [0mapple banana
+[33m[stage-4] [0m[92m✓ Received expected response[0m
+[33m[your-program] [0m$ echo strawberry pear
+[33m[your-program] [0mstrawberry pear
+[33m[stage-4] [0m[92m✓ Received expected response[0m
+[33m[stage-4] [0m[94mWriting history to file using command: history -w /tmp/banana_shell_history_test[0m
```

Comment:
Let's rename history files everywhere to be shorter and include the `.txt` suffix:  (`/tmp/mango.txt`)